polaris-service/src/main/java/org/apache/polaris/service/ratelimiter/ClockImpl.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.polaris.service.ratelimiter;
+
+/** An implementation of our Clock interface using opentelemetry's Clock implementation */
+public class ClockImpl implements Clock {
+  private final io.opentelemetry.sdk.common.Clock openTelemetryClock;
+
+  public ClockImpl() {
+    openTelemetryClock = io.opentelemetry.sdk.common.Clock.getDefault();
+  }
+
+  @Override
+  public long nanoTime() {
+    return openTelemetryClock.nanoTime();
```

Comment:
The only thing OT's clock impl does is calling `System.nanoTime()`.

Reply:
Updated to not depend on the OT clock.